Makefile

```diff
@@ -28,13 +54,25 @@ lint-fix: lint-env
 	golangci-lint run -v --fix ./...
 
 test: test-short
-	go test ${VENDOR} ./...
+	go test $(VENDOR) ./...
 
 test-short:
-	go test ${VENDOR} -race -short
+	go test $(VENDOR) -race -short ./...
 
 build: env
-	CGO_ENABLED=0 go build -ldflags "-X 'bisecur/version.Version=?version?' -X 'bisecur/version.BuildDate=?date?'" -v -o ${DIST}${APPNAME} .
+ifeq ($(OS),Windows_NT)
+	@set CGO_ENABLED=0&& go build -ldflags "$(LDFLAGS)" -v -o "$(OUT)" .
+else
+	CGO_ENABLED=0 go build -ldflags "$(LDFLAGS)" -v -o "$(OUT)" .
+endif
+
+build-linux: env
```

Comment:
If you want to create one Make target per target platform then I propose to create a `build-linux` target and a `build-windows` target too, while pure `build` target would just call both of them. This would make the purpose of the targets clean while we can keep the cross-platform capability you just added. BTW, I tested the current version of the Makefile and it looks okay on Ubuntu too :)